--- configure	Fri Sep 14 02:18:02 2001
+++ configure.in	Fri Feb 22 20:36:00 2002
@@ -1,35 +1,11 @@
 #!/bin/sh
 
 CC=${CC:=cc}
-DEFAULT_HTTPFS_FLAG="0"
-DEFAULT_LIBPATH="/usr/local/lib"
-DEFAULT_BINPATH="/usr/local/bin"
+DEFAULT_HTTPFS_FLAG="1"
+DEFAULT_LIBPATH="/usr/lib"
+DEFAULT_BINPATH="/usr/bin"
 COMPILE_TIME=""
 
-echo -n "Enable the HTTP filesystem support? [y/N]: "
-read HTTPFS_FLAG
-echo -n "Path where to put libnetbrake? [$DEFAULT_LIBPATH]: "
-read LIBPATH
-echo -n "Path where to put the netbrake script? [$DEFAULT_BINPATH]: "
-read BINPATH
-
-[ "$HTTPFS_FLAG" = "y" ] && DEFAULT_HTTPFS_FLAG="1"
-[ "$HTTPFS_FLAG" = "Y" ] && DEFAULT_HTTPFS_FLAG="1"
-[ "$HTTPFS_FLAG" = "yes" ] && DEFAULT_HTTPFS_FLAG="1"
-[ "$LIBPATH" = "" ] || DEFAULT_LIBPATH=$LIBPATH
-[ "$BINPATH" = "" ] || DEFAULT_BINPATH=$BINPATH
-
-echo
-echo ----------------------------------------------
-[ "$DEFAULT_HTTPFS_FLAG" = "1" ] && echo "HTTPFS enabled"
-[ "$DEFAULT_HTTPFS_FLAG" = "0" ] && echo "HTTPFS disabled"
-echo "Library path: $DEFAULT_LIBPATH"
-echo "Binary path: $DEFAULT_BINPATH"
-echo ----------------------------------------------
-echo
-echo "If the configuration is correct press enter, CTRL+C to abort"
-read
-
 [ "$DEFAULT_HTTPFS_FLAG" = "1" ] && COMPILE_TIME="-DNB_HTTPFS"
 
 echo 'Maikefine.in => Makefile ...'
--- libnetbrake.c	2003-12-04 06:33:00.000000000 +0000
+++ libnetbrake.c.new	2003-12-06 03:25:08.000000000 +0000
@@ -51,7 +51,6 @@
 #include <netdb.h>
 #include <time.h>
 #include <locale.h>
-#define _GNU_SOURCE
 #define __USE_GNU
 #include <dlfcn.h>
 
--- Makefile.in.orig	2001-09-10 15:40:50.000000000 +0200
+++ Makefile.in	2004-04-02 22:04:00.784059408 +0200
@@ -37,6 +37,9 @@
 	@$(CC) -MM *.c > .depend
 	@echo Making dependences
 
+libnetbrake.o: libnetbrake.c
+	$(CC) -I. $(CFLAGS) -fPIC $(COMPILE_TIME) $(DEFS) -c $< -o $@
+
 .c.o:
 	$(CC) -I. $(CFLAGS) $(COMPILE_TIME) $(DEFS) -c $< -o $@
 
